Sources/ClientRuntime/Networking/Http/CRT/CRTClientEngine.swift

```diff
@@ -361,4 +419,33 @@ public class CRTClientEngine: HTTPClient {
         response.body = .stream(stream)
         return requestOptions
     }
+
+    actor ContinuationFlag {
+        var continuationResumed = false
```

Comment:
suggestion: can make continuationResume `private` and add a getter to check if it has not been resumed like `setContinuationNotResumed`, that way access to continuationResumed is more explicitly controlled

Reply:
Changed as suggested.